src/main/java/org/example/AccountService.java

```diff
@@ -12,15 +12,23 @@ public Account addAccount(){
         AccountInputService accountInputService = new AccountInputService();
         AccountType accountType = accountInputService.getAccountType();
         CurrencyType currencyType = accountInputService.getCurrency();
+        String password = accountInputService.confirmPassword();
```

Comment:
this function doesn't care about confirmPassword, it should never call confirm password function,
here you need just password and how it gets is the responsibility of AccountInputService.password function
here its OOP concept abstraction